clang/runtime/dpct-rt/include/dpct/codepin/codepin.hpp

```diff
@@ -23,14 +23,27 @@
 #include <unordered_set>
 #include <utility>
 #include <vector>
+#include <stdlib.h>
+
+#ifndef CODEPIN_RAND_SEED
+#define CODEPIN_RAND_SEED 0
+#endif
+
+#ifndef CODEPIN_SAMPLING_THRESHHOLD
+#define CODEPIN_SAMPLING_THRESHHOLD 20
+#endif
+
+#ifndef CODEPIN_SAMPLING_RATE
```

Comment:
codepin-report.py should also contain the sampling infomation, and tells user how to change the sampling info.